src/ReceiptService/index.ts

```diff
@@ -0,0 +1,36 @@
+export class ReceiptService {
+  private static instance: ReceiptService;
+
+  private constructor() {}
+
+  public static getInstance(): ReceiptService {
```

Comment:
The receipt service doesn't need to be singleton. Since CaptureService is a singleton, it can be a property of the capture service.